packages/std/src/math/uint128.rs

```diff
@@ -1,16 +1,19 @@
-use forward_ref::{forward_ref_binop, forward_ref_op_assign};
-use schemars::JsonSchema;
-use serde::{de, ser, Deserialize, Deserializer, Serialize};
 use std::fmt::{self};
 use std::ops::{
     Add, AddAssign, Div, DivAssign, Mul, MulAssign, Rem, RemAssign, Shr, ShrAssign, Sub, SubAssign,
 };
 use std::str::FromStr;
 
+use forward_ref::{forward_ref_binop, forward_ref_op_assign};
+use schemars::JsonSchema;
+use serde::{de, ser, Deserialize, Deserializer, Serialize};
```

Comment:
Did you do this import grouping manually or is there a tool that does this? I'd like to have it that way for the whole codebase.

Reply:
Intellij does this for me automatically via `optimize imports` + the auto formatter. If you want, I could run optimize imports for the whole repo in another PR if you'd like.

Reply:
I see. In this case let's find sone script that does the same thing to make it independent of Intellij.